test/requires/External3.zep

```diff
@@ -14,5 +14,6 @@ class External3
 		ob_clean();
 		require path;
 		requires->setContent(ob_get_contents());
+		ob_clean();
```

Comment:
@Jurigag nah, the test will fail :-) I think PHPUnit is calling ob_clean() after the test and since there is no output buffer any longer, it throws an exception.